packages/front-end/components/Experiment/EditExperimentNameForm.tsx

```diff
@@ -24,6 +24,7 @@ const EditExperimentNameForm: FC<{
       open={true}
       close={cancel}
       size="lg"
+      ctaEnabled={!!form.watch("name").trim()}
```

Comment:
I don't think we should disable submit buttons because it's hard for users to know why. Better to let them try and submit and throw an error

Reply:
ok